ios/RNSScreenContainer.m

```diff
@@ -133,38 +133,41 @@ - (void)updateContainer
   // detect if new screen is going to be activated
   BOOL activeScreenAdded = NO;
   for (RNSScreenView *screen in _reactSubviews) {
-    if (screen.active && ![_activeScreens containsObject:screen]) {
+    if (screen.activityState != 0 && ![_activeScreens containsObject:screen]) {
       activeScreenAdded = YES;
     }
   }
 
+  if (activeScreenRemoved || activeScreenAdded) {
```

Comment:
Do we really need this fragment here? We already have some code for handling "interactionEnabled" prop below that checks the same condition, maybe we can put these fragments next to each other?